public/index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Ramsey Apps 1.0</title>
+    <link rel="apple-touch-icon" sizes="57x57" href="https://cdn.ramseysolutions.net/cms/sites/ramseysolutions-com/favicons/apple-touch-icon-57x57.png">
```

Comment:
🗣️ Why do we have so many of this image in different sizes? apple-touch-icon-120x120.png, 144x144, etc.

Reply:
I just copy pasted this from the mark up on rs.com its for all the different site icons. its not real applicable here but i added it for fun.